internal/bgp/frr/frr_test.go

```diff
@@ -125,6 +126,7 @@ func TestSingleEBGPSessionMultiHop(t *testing.T) {
 			PeerASN:       200,
 			HoldTime:      time.Second,
 			KeepAliveTime: time.Second,
+			ConnectTime:   ptr.To(time.Second),
```

Comment:
Instead of adding it to all the tests, just add it to a couple so we know the default case still works.

Reply:
done :)